content/pt/docs/concepts/components.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Componentes
+description: Os principais componentes que compõem o OpenTelemetry
+aliases: [coleta de dados]
+weight: 20
+default_lang_commit: 3386fa71833d0c25dac410a5db66ae93dc61afcb
+---
+
+O OpenTelemetry é atualmente composto por vários componentes principais:
+
+- [Especificação](#especificação)
+- [Coletor](#coletor)
+- [Implementações de API e SDK específicas](#implementações-de-api--sdk-específicas)
+  - [Bibliotecas de Instrumentação](#bibliotecas-de-instrumentação)
+  - [Exportadores](#exportadores)
+  - [Auto-Instrumentação](#auto-instrumentação)
+  - [Detectores de Recursos](#detectores-de-recursos)
+  - [Propagadores entre Serviços](#propagadores-entre-serviços)
+  - [Amostragen](#amostragem)
+- [Operador Kubernetes](#operador-kubernetes)
+- [Função como Serviço](#função-como-serviço)
```

Comment:
please do not translate anchors, see https://opentelemetry.io/docs/contributing/localization/ for more details

Reply:
@Lindemberg-Chagas aqui explicação sobe ancoras:
https://github.com/edsoncelio/otel-localization-ptbr-guide/blob/main/README.md
![image](https://github.com/user-attachments/assets/c987f42d-9c0b-4a12-9841-666ac3d5aaa7)

Reply:
OK, não sabia o que era esse anchor, vou ver isso

Reply:
Alteração Feita